llms/mlx_lm/LORA.md

```diff
@@ -72,6 +72,27 @@ python -m mlx_lm.lora \
     --test
 ```
 
+### Generate
+
+1. Using the lora module for basic generation:
+
+```
+python -m mlx_lm.lora \
+    --model <path_to_model> \
+    --adapter-file <path_to_adapters.npz> \
+    --max-tokens 50 \
+    --prompt "<your_model_prompt>"
```

Comment:
What do you think about removing `generate` from the original lora script? We could just have it use `generate`. Would simplify things a bit

Reply:
Personally, I feel that using generate.py with an adapter is more natural. hence, it might not be a bad idea to remove generate from the lora script, but this could potentially cause issues for those who are used to fine-tuning testing with lora's generate. Perhaps we could consider starting to raise warnings in lora's generate and removing it in a later version?

Reply:
I don't think many people use the lora generate it so I'm ok just deleting it to simplify the process. I'm also ok with warning for now if you feel otherwise.